### ISSUE TITLE
The order of parameters in sweep config will influence the result

### ISSUE THREAD

**Issue by possible1402** [wandb/examples#82]
I used the sweep example of pytorch version [https://colab.research.google.com/github/wandb/examples/blob/master/colabs/pytorch/Organizing_Hyperparameter_Sweeps_in_PyTorch_with_W%26B.ipynb#](url) and found that if I change the order of parameters, the corresponding will be different. For example, in this example, if the 'dropout' in parameters_dict is changing from [0.3, 0.4, 0.5] to [0.4,0.3,0.5], the results don't match.
![image](https://user-images.githubusercontent.com/29528469/126982328-69aca580-3381-422e-a620-cd21885303e8.png)

**Comment by vanpelt** [comment]
Are you performing a grid search or a random search?  The grid search will just walk through the parameters you've chosen and the order it walks through them will be determined by the order you specified them.  I'm not really sure I understand the question / issue?  Can you try describing the issue more clearly?

**Comment by vanpelt** [comment]
Are you expecting the "loss" values to be the same?  Because each model will start with a seperate random seed even if the hyperparameters are the same the results will be different.

**Comment by tcapelle** [comment]
Did you found a solution on this issue? I will close this as it is not an "Examples" issue and more a wandb/client issue. Feel free to reopen an issue at https://github.com/wandb/client
